presto-hive/src/main/java/com/facebook/presto/hive/parquet/ParquetPageSource.java

```diff
@@ -18,28 +18,35 @@
 import com.facebook.presto.spi.ConnectorPageSource;
```

Comment:
how about a commit message like "Add support for row type in the new Parquet reader"?